types/chai-as-promised/chai-as-promised-tests.ts

```diff
@@ -1,5 +1,5 @@
 import chai = require("chai");
```

Comment:
Does this suggest we should do the same update to the chai types and change this to a default import too afterwards?

Reply:
I don't understand the question :)

chai-as-promised v8 is ESM and wanted to reflect that in the types

Reply:
sorry, what I meant was that we ship chai as ESM now too. But this is still a `require`, so I'm guessing the chai types also need updating

May be worth doing those first so we can update this line to use a default import

Reply:
I am anxiously waiting for types for chai v5. Either as an update in DT or a making chai TypeScript natively...

I thought that the work necessary to do the former is quite big but judgind by this comment https://github.com/chaijs/chai/issues/1554#issuecomment-2190191760, it may not be so bad...